### DISCUSSION TITLE
Consulta persistir usuario en parcial 2canciones

### DISCUSSION THREAD

**Discussion by tomaslcarballo** [dds-jv/foro#331] [Consultas de persistencia] [answered]
Hola, estoy haciendo el parcial 2canciones del ultimo año. Mi duda basicamente es si en este caso hay que persistir al usuario o no ya que no posee ningun atributo, no tiene estado, pero varias clases lo tienen como referencia. Yo en mi caso no lo persistí pero quería consultarles si este camino es el acertado.

![image](https://user-images.githubusercontent.com/62314468/143323870-042393e5-1758-47bc-84d8-480e878d7e4f.png)

Muchas gracias.

Saludos.

**Comment by gastonprieto** [comment, marked as answer]
buenas @tomaslcarballo 

si, hay q persistirlo, probablemente tenga atributos, solo que no lo especificaron, aparte es un concepto importante por su identidad porque identifica al dueño,a los suscriptores, si no lo guardas, entonces no tendrias dueños de canciones ni suscriptores
